validation-test/stdlib/FixedPointDiagnostics.swift.gyb

```diff
@@ -2,13 +2,13 @@
 // RUN: %line-directive %t/main.swift -- %target-swift-frontend -parse -verify %t/main.swift
 
 func testUnaryMinusInUnsigned() {
-  var a: UInt8 = -(1) // expected-error {{}} expected-note * {{}}
+  var a: UInt8 = -(1) // expected-error {{}} expected-note * {{}} expected-warning * {{}}
```

Comment:
I'm not sure I follow. Isn't that warning emitted in a MiscDiagnostics.cpp pass? Are we now getting to that point after emitting an error whereas we didn't used to?